cc-mt-service/src/app.ts

```diff
@@ -1,21 +1,67 @@
-import express, { Express, Request, Response } from "express";
-import dotenv from "dotenv";
+import express, { response } from 'express'
+import axios from 'axios'
+import dotenv from 'dotenv';
 
 dotenv.config();
 
-const app: Express = express();
-const port = process.env.PORT;
+// variables
+const PORT = process.env.PORT;
+const c_id: string = process.env.MT_CLIENTID ?? '';
+const c_secret: string = process.env.MT_CLIENTSECRET ?? '';
+const scope: string = process.env.MT_SCOPE ?? '';
 
-app.use(express.json());
-app.use(
-  express.urlencoded({
-    extended: true,
-  })
-);
 
-app.get("/", (req: Request, res: Response) => {
-  res.send("All data is uploaded to Materialtori :D :D");
+const app = express();
+app.use(express.json());
+app.use(express.urlencoded({extended: true,}));
+app.get("/", (req, res) => {
+  res.send("Adverts will be posted in future");
+});
+app.listen(PORT, () => {
+  console.log(` [server]: Server listening at http://localhost:${PORT}`);
 });
-app.listen(port, () => {
-  console.log(` [server]: Server listening at http://localhost:${port}`);
-});
\ No newline at end of file
+
+showToken(c_id,c_secret,scope);
+/**
+ * Shows Materiaalitori access token in console (just for testing)
+ * @param c_id 
+ * @param c_secret 
+ * @param scope 
+ */
+async function showToken(c_id:string, c_secret:string, scope:string) {
+  const token = await getToken(c_id, c_secret, scope);
+  console.log(token);
+}
+
+/**
+ * Get token to use test.materiaalitori.fi API to make possible to add adverts
+ * @param client 
+ * @param secret 
+ * @param scope 
+ * @returns token to Materiaalitori API
+ */
+async function getToken(client:string, secret:string, scope:string) {
+  try {
+    var axios = require('axios');
```

Comment:
imports from packages should be on top of the file, plus using import syntax and const instead of var is preferred

Reply:
npm run check gives some errors regarding this